### PR TITLE
Release note addition 4.14: Add S2I ga release note item

### PR DESCRIPTION
**Pull request by nebelsprachen**
Version(s):

`4.14`
Issue:

* https://issues.redhat.com/browse/RHDEVDOCS-5614

Link to docs preview:
https://67006--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4.14-builds

QE review:
- [x] QE has approved this change.

Additional information:

SME: [diagrawa](https://issues.redhat.com/secure/ViewProfile.jspa?name=diagrawa) and [rh-ee-apjagtap](https://issues.redhat.com/secure/ViewProfile.jspa?name=rh-ee-apjagtap)
QE: [jitensingh](https://issues.redhat.com/secure/ViewProfile.jspa?name=jitensingh)

### CODE REVIEW

**Comment by joaedwar** [comment]
/enterprise-4.14

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1024,6 +1024,15 @@ With this release, Operator authors can configure their Operator projects with s
 
 For more information, see xref:../operators/operator_sdk/osdk-multi-arch-support.adoc#osdk-multi-arch[Configuring Operator projects for multi-platform support].
 
+[id="ocp-4.14-builds"]
+=== Builds
+
+The Source-to-Image (S2I) tool is now generally available in {product-title} {product-version}. With this update, you can use the S2I tool to build container images from source code, and transform application code into ready-to-deploy container images. This feature enhances the platform's ability to support reproducible containerized application development. 
+
```
Comment (mjpytlak) [inline release_notes/ocp-4-14-release-notes.adoc:1031]: Do the entries for S2I and BuildCSIVolume each require there own heading? Asking, as these seem to be two distinct features being released.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1024,6 +1024,15 @@ With this release, Operator authors can configure their Operator projects with s
...
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_software_collections/2/html/using_red_hat_software_collections_container_images/sti#doc-wrapper[Using Source-to-Image (S2I) tool]. 
+
+BuildCSIVolume is now generally available in {product-title} {product-version}.
```
Comment (mjpytlak) [inline release_notes/ocp-4-14-release-notes.adoc:1034]: Should BuildCSIVolume be Build CSI Volumes? As that is how you have referenced the feature in the Tech Preview table.

**Review by mjpytlak — COMMENTED** [review]
@joaedwar Content looks good, but have a few questions. Please also include a link to the preview build.

Also looks like you still need QE approval.

**Pull request merged**